sdk/storage/Azure.Storage.DataMovement/src/Shared/CheckpointerExtensions.cs

```diff
@@ -102,10 +102,14 @@ internal static void WritePaddedString(this BinaryWriter writer, string value, i
         /// <summary>
         /// Writes a boolean plus int32 to represent a nulable int.
         /// </summary>
-        internal static void Write(this BinaryWriter writer, int? value)
+        internal static void Write(
+            this BinaryWriter writer,
+            int? value,
+            ref int currentOffset)
```

Comment:
All the other write extensions do not increment any sort of offset. Why did you add it to this one specifically? If it needs to stay, we should at least update the docstring to indicate it will increment the offset.

Reply:
Oh I think I added this as something to fix something else, and figured out I didn't need this. I'm going to remove this method all together since this is not being called anywhere.

Thanks for the call out.